styles/style.css

```diff
@@ -0,0 +1,1124 @@
+body {
+  font-family: "Montserrat", sans-serif;
+  margin: 0;
+  padding: 0;
+  color: #555555;
+}
+
+.header {
+  background-image: url("../images/baner.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 22px 190px 140px;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  color: #ffffff;
+}
+
+.header .header-nav {
+  margin-bottom: 182px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+}
+
+.header .header-nav .logo {
+  height: 20px;
+  z-index: 99;
+}
+
```

Comment:
![image](https://user-images.githubusercontent.com/44294945/69324290-dfcea400-0c50-11ea-965e-7cd49d96cff2.png)
there is a diference
![image](https://user-images.githubusercontent.com/44294945/69324318-e9580c00-0c50-11ea-8084-8fd3f5e9da59.png)

1. Layout
2. Your project